Changed old documentation anchors markup for the XREF macro for proper synergy with the downstream documentation (#31548)
quarkusio/quarkus: docs/src/main/asciidoc/hibernate-orm.adoc

Also, Quarkus will set many Hibernate ORM configuration settings automatically, and will often use more modern defaults.

> Please see below section xref:hibernate-configuration-properties[Hibernate ORM configuration properties] for the list of properties you can set in `{config-file}`.

Reviewer: I know it's outside scope, but this sentence needs fixing:

```suggestion
For a list of the items that you can set in `{config-file}`, see xref:hibernate-configuration-properties[Hibernate ORM configuration properties].
```

Author: It definitively does! :D I am on it!

More details about this connection pool can be found in xref:datasource.adoc[Quarkus - Datasources].

Second Level Cache::

> as explained above in section xref:caching[Caching], you don't need to pick an implementation.

Reviewer: ```suggestion
As explained above in section xref:caching[Caching], you don't need to pick an implementation.
```

More details about this connection pool can be found in xref:datasource.adoc[Quarkus - Datasources].

Second Level Cache::

> As explained above in section xref:caching[Caching], you don't need to pick an implementation.

Reviewer: ```suggestion
As explained earlier in the xref:caching[Caching] section, you don't need to pick an implementation.
```

<1> Annotate the TenantResolver implementation with the `@PersistenceUnitExtension` qualifier
to tell Quarkus it should be used in the default persistence unit.
+

> For <xref:multiple-persistence-units[named persistence units], use `@PersistenceUnitExtension("nameOfYourPU")`.

Reviewer 2: ```suggestion
For xref:multiple-persistence-units[named persistence units], use `@PersistenceUnitExtension("nameOfYourPU")`.
```
